.github/workflows/build.yml

```diff
@@ -112,6 +112,30 @@ jobs:
       - name: Run PHPStan
         run: ./vendor/bin/phpstan
 
+  coding-standard:
+    name: Coding standards
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # PHP CS Fixer does not support PHP 8.4 yet
+        php: [8.1, 8.3]
```

Comment:
why would we need to check coding standards on multiple PHP versions ? The formatting of source code won't change depending on the version of PHP being used, so the 2 jobs would report exactly the same thing (and that's a good thing as we cannot change the formatting in different ways if they report different things)